bin/kibana

```diff
@@ -21,4 +21,4 @@ if [ ! -x "$NODE" ]; then
   exit 1
 fi
 
-NODE_OPTIONS="--no-warnings --max-http-header-size=65536 ${NODE_OPTIONS}" NODE_ENV=production exec "${NODE}" "${DIR}/src/cli" ${@}
+NODE_OPTIONS="--max-http-header-size=65536 $(grep -v ^# < ${DIR}/config/node.options | xargs) $NODE_OPTIONS" NODE_ENV=production exec "${NODE}" "${DIR}/src/cli" ${@}
```

Comment:
should max-http-header size be moved too?  iirc it's not a required limit, most common on large dashboards with large query params.

Reply:
I haven't moved `--max-http-header-size=65536` into `node.options` as this will apply also to `kibana-plugin` and `kibana-keystore`. Do you think that would be okay?

Reply:
I see, makes sense.  not bad but not needed.

Reply:
i'm still okay with this on my side, but on my second pass i'm starting to lean towards simplifying.  i'm going to unresolve for more opinions, if not np.

the impact is allowing 64kb headers instead of 8kb in situations where we don't listen over http.  unnecessary, but does let us avoid fragmenting.

Reply:
Yeah I'll be okay with both approaches. I would like to simplify it if there are no concerns around it. What do u think @tylersmalley ?